### PR TITLE
show how to exit

### PR DESCRIPTION
**Pull request by mishfit**
just display how to shutdown the server on startup

### CODE REVIEW

File: src/Microsoft.AspNet.Hosting/Program.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public void Main(string[] args)
             using (host.Start())
             {
                 Console.WriteLine("Started");
+                Console.WriteLine("Press Ctrl+C to shutdown.");
```
Comment (Eilon) [inline src/Microsoft.AspNet.Hosting/Program.cs]: `shut down` is two words.

**Comment by mishfit** [comment]
I don't think that it is (as seen below):
https://en.wikipedia.org/wiki/Shutdown_(computing)
http://www.merriam-webster.com/dictionary/shutdown

node static-server also uses this verbiage.

Do you still want me to change it?

**Comment by Eilon** [comment]
`Shutdown` is a noun, e.g. "the shutdown process." `Shut down` is a compound verb, as in "please shut down the process." You need the verb here.

**Comment by mishfit** [comment]
fair enough.

**Comment by mishfit** [comment]
@eilon done.

**Comment by Eilon** [comment]
@Tratcher look good to you?

**Comment by Tratcher** [comment]
I'd prefer to see the two lines merged: "Application started. Press Ctrl+C to shut down."

**Comment by Eilon** [comment]
Yeah that might be a tiny bit cleaner. @mishfit do you mind making that change? Then we can merge.

**Comment by mishfit** [comment]
@Eilon done.

**Comment by Eilon** [comment]
That was fast! :smile: 

@Tratcher merge please.

**Comment by Tratcher** [comment]
Rebased and merged.

**Pull request closed without merging**